### ISSUE TITLE
Testing and deploying v0.2.0 on OpenShift

### ISSUE THREAD

**Issue by vazirim** [IBM/cloud-operators#189]
- `version: v1beta1` must be added to the descriptions (for service and binding) in the CSV
- add `version: v1beta1` to each of the CRDs
- remove the `replaces: ibmcloud-operator.v0.1.11` in the CSV.
- verify the shape of the deployment specification in the CSV. In particular, it needs to start like:
```
deployments:
      - name: ibmcloud-operator
        spec:
          replicas: 1
          ...
```

More needs to be done.

**Comment by vazirim** [comment]
To debug, go under Workloads on the left hand side of the OpenShift console, and then take a look at Pods. Select the project: `openshift-marketplace` at the top. You will see a pod corresponding to the OperatorSource that was created for the ICO. Navigate to the logs of that pod for more information.

There could be other useful logs to look at, such as the catalog-operator, and marketplace-operator (under `all projects`).

**Comment by JohnStarich** [comment]
Fixed in #200 and in v0.3.0
